requirements-dev.txt

```diff
@@ -7,8 +7,8 @@ numpy>=1.14
 # Optional dependencies in Koalas.
 mlflow>=1.0
 
-# Documentation build
-sphinx<3.1.0
+# Documentation build.
+sphinx>=2.0.0,<3.1.0
```

Comment:
The lowerbound is set because:
- The documentation style become slightly changed when we use 1.8. 
- Read the Docs environment has Sphinx 1.8 by default. It should better upgrade.